.azure-pipelines/master.yml

```diff
@@ -83,6 +83,42 @@ stages:
               ArtifactName: 'osx-64'
             displayName: 'Publish Conda package artefacts'
 
+      - job: 'windows'
+        displayName: 'Windows'
+        timeoutInMinutes: 120
```

Comment:
Quite a long time-out. Is this necessary, was this what we had in place before? For other platforms we are timing out at most at 90 minutes.

Reply:
This value is unchanged from the old windows CI. I would assume it is higher because the windows CI builds can take significantly longer than those on the other platforms.